packages/components/label/src/label.tsx

```diff
@@ -59,6 +61,17 @@ type TLabelProps = {
 };
 
 const Label = (props: TLabelProps) => {
+  const { theme } = useTheme();
+  // TODO: This is a temporary solution due to theme migration. After the new
+  // theme is published, we must remove this and just use the `Text.Detail` component
+  const TextComponent = theme === 'default' ? Text.Body : Text.Detail;
+
+  const getLabelStyles = () => css`
+    > div {
+      font-weight: 500;
+    }
+  `;
```

Comment:
I think we shouldn't hardcode values and use a design token instead.
I suppose we don't need to define as a function, but if that is preferred then not define it inside the component.

Reply:
I wasn't really sure how to implement this change and if my implementation in correct. Any other suggestion is more than welcome! 😄